src/components/Match/Overview/PicksBans.jsx

```diff
@@ -70,21 +66,26 @@ img {
 }
 `;
 
-const PicksBans = ({ data, strings, style }) => (
-  <Styled style={style}>
-    <div className="PicksBans">
-      {data.map(pb => (
-        <section key={pb.order}>
-          <HeroImage id={pb.hero_id} imageSizeSuffix={IMAGESIZE_ENUM.SMALL.suffix} data-isPick={pb.is_pick} />
-          {!pb.is_pick && <div className="ban" />}
-          <aside>
-            {pb.is_pick ? strings.match_pick : strings.match_ban} <b>{pb.order + 1}</b>
-          </aside>
-        </section>
-      ))}
-    </div>
-  </Styled>
-);
+
+const PicksBans = ({ gameMode, data, strings, style }) => {
+  let counter = 0;
```

Comment:
what's this for?

Reply:
Whoops, that is supposed to be the ban number, renamed now. I just added a variable because I could not think of a more elegant way to do it